core/core-impl/src/main/java/ai/timefold/solver/core/config/solver/monitoring/SolverMetric.java

```diff
@@ -36,7 +36,10 @@ public enum SolverMetric {
     PICKED_MOVE_TYPE_BEST_SCORE_DIFF("timefold.solver.move.type.best.score.diff", new PickedMoveBestScoreDiffStatistic<>(),
             true),
     PICKED_MOVE_TYPE_STEP_SCORE_DIFF("timefold.solver.move.type.step.score.diff", new PickedMoveStepScoreDiffStatistic<>(),
-            false);
+            false),
+    PROBLEM_ENTITY_COUNT("timefold.solver.problem.entities", false),
+    PROBLEM_VARIABLE_COUNT("timefold.solver.problem.variables", false),
+    PROBLEM_VALUE_COUNT("timefold.solver.problem.values", false);
```

Comment:
Since these metrics are enabled by default, I'd put them amongst the others which are enabled by default, at the top. The enum is not inside public API, so I don't worry about ordinals staying constant.